<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OTables</class>
 <widget class="QMainWindow" name="OTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ООО &quot;Варнофф&quot;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fcfcfc;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="Russian" country="Russia"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
    <property name="spacing">
     <number>22</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">background-color: #fcfcfc;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,2,2,2,2,2,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="labelSearch">
          <property name="text">
           <string>Поиск:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearch">
          <property name="styleSheet">
           <string notr="true">border-color: #1a1a1a;
border: 1px solid;

background-color: transparent;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonContacts">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
        </property>
        <property name="text">
         <string>Контакты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCompanies">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
        </property>
        <property name="text">
         <string>Компании</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInfo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
        </property>
        <property name="text">
         <string>Инфо о компании</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonServices">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
        </property>
        <property name="text">
         <string>Услуги</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonContracts">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
        </property>
        <property name="text">
         <string>Контракты</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEntry">
    <property name="styleSheet">
     <string notr="true">QMenu::item:selected {
    background-color: #c9c9c9;
	color: #000
}</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="title">
     <string>Запись</string>
    </property>
    <addaction name="actionAddEntry"/>
    <addaction name="actionEditEntry"/>
    <addaction name="actionDelEntry"/>
   </widget>
   <widget class="QMenu" name="menuTable">
    <property name="styleSheet">
     <string notr="true">QMenu::item:selected {
    background-color: #c9c9c9;
	color: #000
}</string>
    </property>
    <property name="title">
     <string>Таблица</string>
    </property>
    <addaction name="actionImportTable"/>
    <addaction name="actionExportTable"/>
   </widget>
   <widget class="QMenu" name="menuSep">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>|</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
   </widget>
   <addaction name="menuEntry"/>
   <addaction name="menuTable"/>
   <addaction name="menuSep"/>
  </widget>
  <action name="actionAddEntry">
   <property name="text">
    <string>Добавить...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionEditEntry">
   <property name="text">
    <string>Изменить...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDelEntry">
   <property name="text">
    <string>Удалить...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionImportTable">
   <property name="text">
    <string>Импорт...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExportTable">
   <property name="text">
    <string>Экспорт...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
